tests/phpunit/integration/api/html/beansRemoveOutput.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Tests for beans_remove_output().
+ *
+ * @package Beans\Framework\Tests\Integration\API\HTML
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Integration\API\HTML;
+
+use _Beans_Anonymous_Filters;
+use Beans\Framework\Tests\Integration\API\HTML\Includes\HTML_Test_Case;
+
+require_once __DIR__ . '/includes/class-html-test-case.php';
+
+/**
+ * Class Tests_BeansRemoveOutput
+ *
+ * @package Beans\Framework\Tests\Integration\API\HTML
+ * @group   api
+ * @group   api-html
+ */
+class Tests_BeansRemoveOutput extends HTML_Test_Case {
+
+	/**
+	 * Test beans_remove_output() should return an _Beans_Anonymous_Filters instance.
```

Comment:
`return a _Beans_Anonymous_Filters instance.`
(or is the underscore pronounced?)